### PR TITLE
D band ohm

### PR DESCRIPTION
**Pull request by MakerPact**
Lets try this again.

### CODE REVIEW

**Comment by MakerPact** [comment]
Added tags: Ohms law, and De-bounce. Also corrected spelling error. Requested by helpers. :D

File: src/tags/debounce.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+      "name": "Hardware debounce:",
+      "value": "If you use a resistor and a capacitor, you can filter out the micro contacts of most buttons and or switches. You need to select the correct capacitor, and the correct resistor for the job. This can make wiring a bit more complex as your adding 4 connections."
+    },
+    {
+      "name": "Software debounce through code.",
+      "value": "We can also use code to debounce the button/switch. You can go you your arduino IDE. File>Examples>Digital>Debounce. You can find the code here if you want an example that you can use to incorporate your code into it."
+    }
+  ],
+  "image": "https://ibb.co/rsgLjMH",
+  "image": "http://wiki.ros.org/rosserial_arduino/Tutorials/Push%20Button?action=AttachFile&do=get&target=debounce.png"
```
Comment (max-bromberg) [inline src/tags/debounce.json]: There can only be one image associated with a tag. Please choose one to keep.

**Review by max-bromberg — CHANGES_REQUESTED** [review]
Please see specific comments.

File: src/tags/debounce.json (hunk 1 above)
Comment (MakerPact) [reply src/tags/debounce.json]: The extra image should be removed.

**Review by max-bromberg — APPROVED** [review]
Looks good!

**Comment by max-bromberg** [comment]
Thanks for your contribution!

**Pull request merged**